app/services/api/webflow-api.ts

```diff
@@ -112,17 +127,17 @@ export const useScheduleScreenData = () => {
       {
         date: "2023-05-17",
         title: "React Native Workshops",
-        events: convertScheduleToScheduleCard(events, "Wednesday"),
+        events: events ? convertScheduleToScheduleCard(events, "Wednesday") : [],
```

Comment:
App blew up during a loading state when `isLoading` is true.

Reply:
![Simulator Screen Shot - iPhone 13 - 2023-04-13 at 21 15 21](https://user-images.githubusercontent.com/37849890/231940179-5cec502e-20e1-4525-95bf-9b3a15bc1b37.png)